.sphinx/.wordlist.txt

```diff
@@ -280,3 +281,9 @@ Zettabyte
 ZFS
 zpool
 zpools
+AMD
+SEV
+ES
+Diffie
+Hellman
+base64
```

Comment:
@ru-fu can you go through this commit and see which ones actually deserve an exception and which ones just need quoting?